packages/plugin-npm-cli/sources/commands/npm/info.ts

```diff
@@ -17,130 +17,135 @@ declare module '@npm/types' {
   }
 }
 
+/**
+ * The combined type of `Packument` (without the `versions` field) and `PackumentVersion`
+ */
+type CombinedPackument = Omit<npm.Packument, 'versions'> & npm.PackumentVersion;
+
+/**
+ * `CombinedPackument` with a `versions` field that is an array of tags
+ */
+interface PackageInformation extends CombinedPackument {
+  versions: string[];
+}
+
 // eslint-disable-next-line arca/no-default-export
 export default class InfoCommand extends BaseCommand {
-  @Command.String({required: true})
-  package!: string;
-
-  @Command.String({required: false})
-  field!: string;
+  @Command.Rest()
+  packages!: string;
 
-  @Command.Boolean(`-v,--verbose`)
-  verbose: boolean = false;
+  @Command.String(`-f,--fields`)
+  fields?: string;
 
   @Command.Boolean(`--json`)
   json: boolean = false;
 
   static usage: Usage = Command.Usage({
     description: `show information about a package`,
     details: `
-      This command will fetch information about a package and return it in a tree format.
-      The package does not have to have been installed locally.
+      This command will fetch information about a package from the npm registry, and prints it in a tree format.
 
-      The default reporting style for this command is a single-quoted serialization (JSON5). To emit lines of valid JSON, use the \`--json\` flag.
+      The package does not have to be installed locally, but needs to have been published (in particular, local changes will be ignored even for workspaces).
 
-      Append \`@[version]\` to the package argument to provide information specific to that version. If the version is invalid, the command falls back to latest.
+      Append \`@<version>\` to the package argument to provide information specific to that version. If the version is invalid, the command will print a warning and fall back to latest.
```

Comment:
The command doesn't currently print a warning if the version is invalid.

Reply:
This is something I can implement, but only in a few hours.

Reply:
Sounds good! No rush 👍